requirements/app/base.txt

```diff
@@ -7,3 +7,4 @@ s3fs>=2022.5.0, <=2022.7.1
 croniter>=1.3.0, <1.4.0  # strict; TODO: for now until we find something more robust.
 traitlets<5.2.0 # Traitlets 5.2.X fails: https://github.com/ipython/traitlets/issues/741
 arrow>=1.2.0, <=1.2.2
+lightning-utilities>=0.2.0
```

Comment:
Let's pin a specific version or range... 
```suggestion
lightning-utilities==0.2.*
```

Reply:
Thanks. I'll do this in a follow-up to avoid conflicts with the other PRs. I would like to avoid re-running CI if possible